RandomPractice.py

```diff
@@ -0,0 +1,40 @@
+import random
+
+#define a function to randomly generate a given length string
+def generateOne(n):
+    alphabet="abcdefghijklmnopqrstuvwxyz "
+    return random.choices(alphabet,k=n)
+#the first method returns a list of letters
+
+def generateTwo(n):
+    alphabet="abcdefghijklmnopqrstuvwxya "
+    teststring=""
+    for i in range(n):
+        teststring=teststring+alphabet[random.randrange(len(alphabet))]

```

Comment:
can be written as:
`"".join( (alphabet[random.randrange(len(alphabet))] for i in range(n) ) )`

Reply:
brilliant! haven't thought of using list comprehension in such circumstances. but you use tuple instead of list?

Reply:
good question, but this is not tuple.